### PR TITLE
1842 Store user session in shared keychain

### PR DESCRIPTION
**Pull request by zamorastp**

### CODE REVIEW

File: Sources/AccountSDKIOSWeb/User/User.swift [hunk 1]
```diff
@@ -328,3 +330,23 @@ extension User {
     }
 
 }
+
+// MARK: Shared Keychain
+
+extension User {
+    public func storeSessionInSharedKeychain(appleTeamId: String, completion: @escaping (Result<Void, Error>) -> Void) {
```
Comment (Daniel-Echegaray) [inline Sources/AccountSDKIOSWeb/User/User.swift:337]: I am thinking if this should be public, or if the SDK should handle storing to shared keychain internally. 
How is the intended use case for the brands?

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/User/User.swift:337]: In my understanding, shared keychain entitlement should be switched on with the correct group to allow saving
 data.

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/User/User.swift:337]: So at least we need public method to switch it on?

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/User/User.swift:337]: Also i am going to check how we can get teamID programatically

**Pull request closed without merging**